Data-Analytics/Spark-GPU/Python/scala-gpu-example.yaml

```diff
@@ -14,13 +14,13 @@ spec:
     spark.task.resource.gpu.amount: "1"
     spark.executor.resource.gpu.amount: "1"
     spark.executor.resource.gpu.vendor: "nvidia.com"
-    spark.executor.resource.gpu.discoveryScript: "/opt/mapr/spark/spark-3.5.0/examples/src/main/scripts/getGpusResources.sh"
-    spark.rapids.shims-provider-override: "com.nvidia.spark.rapids.shims.spark350.SparkShimServiceProvider"
+    spark.executor.resource.gpu.discoveryScript: "/opt/mapr/spark/spark-3.5.1/examples/src/main/scripts/getGpusResources.sh"
+    spark.rapids.shims-provider-override: "com.nvidia.spark.rapids.shims.spark351.SparkShimServiceProvider"
  
   type: Python
-  sparkVersion: 3.5.0
+  sparkVersion: 3.5.1
   mode: cluster
-  image: gcr.io/mapr-252711/spark-gpu-3.5.0:v3.5.0
+  image: gcr.io/mapr-252711/spark-gpu-3.5.1:v3.5.1.0.0
   imagePullPolicy: Always
   mainApplicationFile: "local:///mounts/shared-volume/ezua-tutorials/current-release/Data-Analytics/Spark-GPU/Python/SqlOnGpuExample.py"
```

Comment:
Can we correct the file path to local:///mounts/shared-volume/shared/ezua-tutorials